src/env.h

```diff
@@ -708,8 +741,14 @@ class Environment {
 
   double* heap_statistics_buffer_ = nullptr;
   double* heap_space_statistics_buffer_ = nullptr;
+  uint32_t* http2_settings_buffer_ = nullptr;
+  uint32_t* http2_options_buffer_ = nullptr;
+  double* http2_session_state_buffer_ = nullptr;
+  double* http2_stream_state_buffer_ = nullptr;
+  uint32_t* http2_padding_buffer_ = nullptr;
```

Comment:
It seems like all of these have static sizes, why not just make them static arrays?